test-utils/src/dev/mod.rs

```diff
@@ -118,13 +132,14 @@ pub async fn test_setup_database(
     source: StorageSource,
 ) -> db::CockroachInstance {
     usdt::register_probes().expect("Failed to register USDT DTrace probes");
-    setup_database(log, source).await
+    setup_database(log, source).await.unwrap()
 }
 
+// TODO: switch to anyhow entirely -- this function is currently a mishmash
```

Comment:
Just to clarify, this appears to already be using an `anyhow::Result`?

Reply:
So this is currently a mishmash of `anyhow::Result` and `unwrap`/`expect`. I've expanded the comment to make this clearer.